lightning/src/ln/channelmanager.rs

```diff
@@ -791,6 +787,22 @@ pub struct ChannelManager<M: Deref, T: Deref, K: Deref, F: Deref, L: Deref>
 	/// See `ChannelManager` struct-level documentation for lock order requirements.
 	id_to_peer: Mutex<HashMap<[u8; 32], PublicKey>>,
 
+	/// SCIDs (and outbound SCID aliases) -> `counterparty_node_id`s and `channel_id`s.
+	///
+	/// Outbound SCID aliases are added here once the channel is available for normal use, with
+	/// SCIDs being added once the funding transaction is confirmed at the channel's required
+	/// confirmation depth.
+	///
+	/// Note that while this holds `counterparty_node_id`s and `channel_id`s, no consistency
+	/// guarantees are made about the existence of a peer with the `counterparty_node_id` nor a
+	/// channel with the `channel_id` in our other maps.
+	///
+	/// See `ChannelManager` struct-level documentation for lock order requirements.
```

Comment:
nit: Making `ChannelManager` a doc link would be helpful to get straight to the struct-level docs.

Reply:
Ah I thought it was the case that private fields don't appear on the docs website therefore links wouldn't be clickable anywhere?

Reply:
Oh whoops. Didn't catch this was private :)